libs/domain/order/src/services/order.routes.ts

```diff
@@ -1,24 +1,9 @@
-import { resolve } from '@spryker-oryx/di';
 import { RouteType } from '@spryker-oryx/router';
 import { RouteConfig } from '@spryker-oryx/router/lit';
-import { combineLatest, map, take } from 'rxjs';
-import { OrderService } from './order.service';
 
 export const orderRoutes: RouteConfig[] = [
   {
     path: '/order/:id',
     type: RouteType.Order,
-    enter: ({ id }) =>
-      combineLatest([
```

Comment:
Why did we drop it?
In this case we don't have gurad for incorrect order

Reply:
It doesn't work in SSR because it needs session data from the client side, so we always end up showing 404 pages except directly after checkout.

Discussed it with Stan, and he said to remove it for now. Feel free to ask him more about it.